xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -79,7 +79,7 @@ void CGUIWindowPictures::OnInitWindow()
     {
       if (wndw && wndw->GetCurrentSlide())
         m_viewControl.SetSelectedItem(wndw->GetCurrentSlide()->GetPath());
-      m_iSelectedItem = m_viewControl.GetSelectedItem();
+      SetMediaSelectedItem(m_vecItems->GetPath(),m_viewControl.GetSelectedItem());
```

Comment:
Because right now there's only one selected item stored. If you go to movie and navigate to the 4th movie and then you go to tv shows you'll get the 4th tv show selected at start instead of starting from the beginning